plugins/destination/duckdb/client/transform_gotype.go

```diff
@@ -42,6 +43,20 @@ func getTypedNilValue(arr arrow.Array) any {
 		return nilPtrOf[string]()
 	case *arrow.BinaryType, *arrow.LargeBinaryType, *arrow.FixedSizeBinaryType:
 		return nilPtrOf[[]byte]()
+	case *arrow.ListType:
```

Comment:
can we use [`arrow.ListLikeType`](https://pkg.go.dev/github.com/apache/arrow/go/v15@v15.0.0/arrow#ListLikeType) interface to match both lists (NB: also matches maps)?

Reply:
yeah it matches maps as well, but we can order it carefully

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/16668/commits/3ce23706feccf97e67ba6820483d56044115844d

Reply:
I still see it here, maybe it was done only in the values but not in nils?